lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -123,7 +123,12 @@ Function Join-Domain {
 
     if($domain_ou_path){
         Write-DebugLog "adding OU destination arg to Add-Computer args"
-        $add_args["OUPath"] = $domain_ou_path
+        if([System.DirectoryServices.DirectoryEntry]::Exists("LDPA://$domain_ou_path")){
+            $add_args["OUPath"] = $domain_ou_path
+        }
+        else {
+            Throw "The specified OU $domain_ou_path was not found."
```

Comment:
Best to just call `Fail-Json -obj $result -message "The specified OU '$domain_out_path' was not found." instead of throwing the exception.

Reply:
Ok, thank you for the advice